composer.json

```diff
@@ -11,7 +11,17 @@
     ],
     "require": {
         "php": "^7.1",
-        "laravel/framework": "~5.7"
+        "illuminate/support": "~5.7",
```

Comment:
Don't you get `illuminate/support` as subtree split of Support component within `laravel/framework` (which is already required)?

Reply:
Yes, you're right! 👍